server/src/main/java/com/lunchtime/seed/DatabaseSeed.java

```diff
@@ -18,8 +18,7 @@
 public class DatabaseSeed {
 
     String[] dishesName = new String[] {"Salami", "Margarita", "Manhattan", "Soup", "Hamburger", "Ice"};
-    String[] categoryFood = new String[] {"Pizza", "Pizza", "Pizza",
-                                          "Main course", "Snacks", "Dessert"};
+    String[] categoryFood = new String[] {"Pizza", "Pizza", "Pizza", "Main course", "Snacks", "Dessert"};
```

Comment:
Why you add 3 "Pizza"? This file generates in fresh start, you should remove redundant categories

Reply:
This code is on dev and in steging server. I have added here one "Pizza" to category but it works not  with DB created on dev. It don't shows any dishes in menu. I think it will be happen on staging server to.  Therefore, I marked the extra fields in the database as isDeleted "true". I have merged dev to my branch and it works as well.  
On staging server we will not create new DB. It will be added to existing DB, that 2 extra fields in categoryfood table are maked as isDeleted. Is that supposed to work that way?

Reply:
This code invokes only when fresh project starts, it doesn't affect already created database. You should remove unnecessary items from code

Reply:
I have deleted, it works, thank you.